api/controllers/users.py

```diff
@@ -217,9 +217,12 @@ def recover_password():
             "ui_server_host": util.parse_url(bottle.request.url),
             "token": forgot_password_token,
         }
+        logger.info("Using default bottle app config for mail config")
```

Comment:
I don't think this log message is necessary

Reply:
removed